recipes-kernel/linux/files/patches-5.10/0123-HACK-kbuild-generate-__symbol_-node-in-the-dtb-file.patch

```diff
@@ -0,0 +1,30 @@
+From 0d7b443c67bfeaaed6e0d768c10584fd4a3d36d2 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Fri, 14 Jan 2022 14:22:14 +0800
+Subject: [PATCH 123/123] HACK: kbuild: generate __symbol_ node in the dtb file
+
+add the build -@ to generate __symbol__ node, lacking this
+node,dtbo can not apply to the base dtb.
```

Comment:
Still a hack, we need to fix this.

Reply:
Better look at https://github.com/torvalds/linux/commit/e426d63e752bdbe7d5ba2d872319dde9ab844a07 and add this when adding the overlay dts files to the kernel.